From 38a24392f83b3b33012fae851711c0367df95d0c Mon Sep 17 00:00:00 2001
From: William Wu <william.wu@rock-chips.com>
Date: Sat, 6 Oct 2018 20:13:07 +0800
Subject: [PATCH] arm64: dts: rockchip: set dwc3 in gadget only mode for rk1808

The dwc3 controller on rk1808 evb with Type-A interface can
act as static host and static device. However, the dwc3 driver
can't switch host and device mode well, so we set dwc3 in gadget
only mode for the time being.

Change-Id: Iaa2af5791dacecdd1e2c784d50f0fea6b3a9deb4
Signed-off-by: William Wu <william.wu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808.dtsi b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
index c43094230564..d3383203e8ad 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
@@ -112,7 +112,7 @@
 			compatible = "snps,dwc3";
 			reg = <0x0 0xfd000000 0x0 0x200000>;
 			interrupts = <GIC_SPI 75 IRQ_TYPE_LEVEL_HIGH>;
-			dr_mode = "otg";
+			dr_mode = "peripheral";
 			phys = <&u2phy_otg>;
 			phy-names = "usb2-phy";
 			phy_type = "utmi_wide";
-- 
2.35.3

